### PR TITLE
[ADD][10.0] website_sale_show_company_data: New Module

### PR DESCRIPTION
**Pull request by chienandalu**
When portal users that belong to a company edit their invoicing address during
the checkout proccess there is no clue whether the order is being billed to
their company or to the user themselves.

This module adds the company name and VAT info in readonly mode to give a
reference to the user.

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This has nothing to do with `portal` modules, so you should rename it and simplify to `website_sale_company_data`, and put on the roadmap other possible places where this can be changed in the checkout process.

**Comment by pedrobaeza** [comment]
Check also Travis status.

**Comment by rafaelbn** [comment]
@crismartinro could you please review?

**Comment by chienandalu** [comment]
Maybe you meant to ping @cristinamartinrod?

File: website_sale_show_company_data/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 - Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Portal Show Company Data",
+    "summary": "Show company partner data if any",
```
Comment (pedrobaeza) [inline website_sale_show_company_data/__manifest__.py]: s/company/commercial

File: website_sale_show_company_data/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 - Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Portal Show Company Data",
```
Comment (pedrobaeza) [inline website_sale_show_company_data/__manifest__.py]: Remove `Portal` word and regenerate README

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Review by cristinamartinrod — APPROVED** [review]
Works fine!

**Pull request merged**